linkcheck/logger/customxml.py

```diff
@@ -48,20 +47,20 @@ def log_url(self, url_data):
         """
         self.xml_starttag('urldata')
         if self.has_part('url'):
-            self.xml_tag("url", str_text(url_data.base_url))
+            self.xml_tag("url", url_data.base_url)
```

Comment:
These are strings, not URL objects with a `__str__`, right?

Reply:
The are strings.